src/index.html

```diff
@@ -3,11 +3,320 @@
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header">
+      <div class="header__top top-bar">
+        <a href="#">
+          <img
+            class="top-bar__logo"
+            src="images/logo.png"
+            alt="nothing logo"
+          >
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="#" class="icon icon--phone"></a>
+
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+      </div>
+      <div class="header__bottom">
+        <h1 class="header__title">Bring joy back<br> to the everyday</h1>
+      </div>
+    </header>
+
+    <aside class="menu__sidebar" id="menu">
+      <div class="menu">
+        <div class="menu__top top-bar">
+          <a href="#">
+            <img
+              class="top-bar__logo"
+              src="images/logo.png"
+              alt="nothing logo"
+            >
+          </a>
+
+          <div class="menu__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+
+        <div class="menu__content">
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="list__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Products
+                </a>
+              </li>
+
+              <li class="list__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Store
+                </a>
+              </li>
+
+              <li class="list__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Login
+                </a>
+              </li>
+
+              <li class="list__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                Cart(0)
+                </a>
+              </li>
+            </ul>
+          </nav>
+
+          <div class="menu__contacts">
+            <a class="menu__contacts--link" href="tel:+123455555555">+1 234 5555-55-55</a>
+
+            <a class="menu__contacts--link menu__contacts--link--active" href="tel:+123455555555">call to order</a>
+          </div>
+        </div>
+      </div>
+
+    </aside>
+
+    <main class="main">
+      <div class="main__container">
+        <div class="main__content">
+          <section class="recommended" id="recommended">
+            <h2 class="recommended-title">Recommended</h2>
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <a href="#" class="product__link">
+                  <img
+                  src="./images/product/phone-1.png"
+                  alt="Phone (1)"
+                  class="product__img"
+                >
```

Comment:
![image](https://github.com/mate-academy/layout_landing-page/assets/54860641/83985804-fcad-4217-8110-51d21945590e)
Fix this in all code.